<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvToolCropImage</class>
 <widget class="QWidget" name="vvToolCropImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="xminSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spin_xmin"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="xmaxSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spin_xmax"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="yminSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spin_ymin"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y max</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="ymaxSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spin_ymax"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mZLabel1">
       <property name="text">
        <string>Z min</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="zminSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spin_zmin"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mZLabel2">
       <property name="text">
        <string>Z max</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="zmaxSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spin_zmax"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_xmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xminSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xminSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_xmin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xmaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_xmax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yminSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_ymin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ymaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_ymax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zminSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_zmin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zmaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_zmax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_xmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xmaxSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_ymin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yminSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_ymax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ymaxSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_zmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zminSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_zmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zmaxSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
